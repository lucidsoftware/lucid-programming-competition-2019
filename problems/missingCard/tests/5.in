25 5
7 25 13 2 5 19 9 18 21 14 10 8 11 12 23 16 1 24 22 17 6 3 20 15
1 20 15 17 13 11 8 9 16 19 3 23 2 14 21 24 4 12 6 5 10 22 18 25
16 8 22 17 24 23 6 9 20 2 15 10 14 5 18 3 19 21 4 7 25 11 13 12
3 15 23 6 12 7 11 19 25 17 9 14 22 16 20 5 10 1 18 4 21 13 2 8
24 16 18 1 9 17 5 13 22 11 14 6 2 21 8 10 7 23 19 3 20 12 25 4