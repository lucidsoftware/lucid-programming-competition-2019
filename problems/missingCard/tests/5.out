4
7
1
24
15